app/build.gradle

```diff
@@ -97,6 +99,7 @@ dependencies {
         exclude module: 'recyclerview-v7'
     }
     annotationProcessor 'androidx.room:room-compiler:2.2.6'
+    implementation 'de.hdodenhof:circleimageview:3.1.0'
```

Comment:
what is that ?

Reply:
This is an implementation for having rounded header pictures in ActivityDescriptionActvity. This has been added by Jean-Luc!